Paymetheus.Decred/Wallet/WalletSeed.cs

```diff
@@ -29,7 +29,16 @@ public static byte[] DecodeAndValidateUserInput(string userInput, PgpWordList pg
                 throw new ArgumentNullException(nameof(pgpWordList));
 
             var decodedInput = DecodeUserInput(userInput, pgpWordList);
-            if (decodedInput.Length != SeedLength)
+            if (decodedInput.Length == (SeedLength + 1))
+            {
+                // deal with "checksum"
+                var digest = Checksum.DoubleSha256(decodedInput);
+                if (decodedInput[SeedLength] != digest[0])
+                {
+                    throw new Exception($"Invalid checksum");
```

Comment:
The `$""` string is for string interpolation.  Since this is not formatting any values in the string, you can just use normal quotes `""`.